### PR TITLE
bootstrap: use gopkg.in import for square/go-jose

### PR DESCRIPTION
**Pull request by ericchiang**
xref #55514

For 1.10. Ignore while 1.9 code freeze is in effect.

```release-note
NONE
```

### CODE REVIEW

File: hack/update-godep-licenses.sh [hunk 1]
```diff
@@ -75,6 +75,9 @@ process_content () {
     go4.org/*)
      package_root=$(echo ${package} |awk -F/ '{ print $1 }')
      ;;
+    gopkg.in/*)
```
Comment (thockin) [inline hack/update-godep-licenses.sh]: The LICENSES diff is impossible to read.  Can I get you to send a PR with *just* this change (and licenses updated), so I can verify that piece of it?

Comment (ericchiang) [reply hack/update-godep-licenses.sh]: Sent https://github.com/kubernetes/kubernetes/pull/56231

**Comment by thockin** [comment]
Is this "fixes" or "xref" the issue above?

**Comment by ericchiang** [comment]
xref. Will edit the description

**Comment by ericchiang** [comment]
/hold

https://github.com/kubernetes/kubernetes/pull/56231 should merge first.

File: pkg/controller/bootstrap/jws.go [hunk 2]
```diff
@@ -20,19 +20,28 @@ import (
...
 // is a regular JWS with the content part of the signature elided.
 func computeDetachedSig(content, tokenID, tokenSecret string) (string, error) {
-	jwk := &jose.JsonWebKey{
+	jwk := &jose.JSONWebKey{
 		Key:   []byte(tokenSecret),
 		KeyID: tokenID,
 	}
 
-	signer, err := jose.NewSigner(jose.HS256, jwk)
+	opts := &jose.SignerOptions{
```
Comment (ericchiang) [inline pkg/controller/bootstrap/jws.go]: New behavior ¯\_(ツ)_/¯

Comment (dims) [reply pkg/controller/bootstrap/jws.go]: who is jose? :)

Comment (dims) [reply pkg/controller/bootstrap/jws.go]: (j/k)

**Pull request merged**